dev/CommandBarFlyout/CommandBarFlyout.xaml

```diff
@@ -323,9 +323,8 @@
                         <Grid x:Name="OuterContentRoot"
                             VerticalAlignment="Top"
                             Margin="{TemplateBinding Padding}"
-                            Width="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=FlyoutTemplateSettings.CurrentWidth}"
+                            MinWidth="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=FlyoutTemplateSettings.CurrentWidth}"
                             Height="{TemplateBinding Height}"
-                            Background="{TemplateBinding Background}"
```

Comment:
>t [](start = 47, length = 1)

Why did the background have to get moved? can it be moved to the ContentRoot grid instead of both its children?

Reply:
Oh forgot to reply, the reason why we moved the background is that the "outer" container is not lined up with the animation of the button. The width lags behind resulting in the background being visible even if the button already animated to the correct position. Updated it to use the contentroot grid instead.